lib/rules/no-duplicate-selectors/index.mjs

```diff
@@ -1,12 +1,14 @@
-import resolvedNestedSelector from 'postcss-resolve-nested-selector';
-import selectorParser from 'postcss-selector-parser';
-
 import findAtRuleContext from '../../utils/findAtRuleContext.mjs';
+import flattenNestedSelectorsForRule from '../../utils/flattenNestedSelectorsForRule.mjs';
+import getRuleSelector from '../../utils/getRuleSelector.mjs';
+import getSelectorSourceIndex from '../../utils/getSelectorSourceIndex.mjs';
 import { isBoolean } from '../../utils/validateTypes.mjs';
 import isKeyframeRule from '../../utils/isKeyframeRule.mjs';
-import isStandardSyntaxSelector from '../../utils/isStandardSyntaxSelector.mjs';
+import isStandardSyntaxRule from '../../utils/isStandardSyntaxRule.mjs';
 import nodeContextLookup from '../../utils/nodeContextLookup.mjs';
+import normalizeSelector from '../../utils/normalizeSelector.mjs';
 import parseSelector from '../../utils/parseSelector.mjs';
+import parser from 'postcss-selector-parser';
```

Comment:
[suggest] Can you move importing `postcss-selector-parser` to the top of this file? I want to distinguish between importing outside packages and inside modules.

Also, `parser` is ambiguous, so I'd like to rename it. For example:

```suggestion
import parserSelector from 'postcss-selector-parser';
```